src/cli/client.rs

```diff
@@ -331,6 +332,32 @@ impl KrillClient {
                 Ok(ApiResponse::BgpAnalysisSuggestions(suggestions))
             }
 
+            CaCommand::AspasList(handle) => {
+                let uri = format!("api/v1/cas/{}/aspas", handle);
+                let aspas = get_json(&self.server, &self.token, &uri).await?;
+                Ok(ApiResponse::AspaDefinitions(aspas))
```

Comment:
Same question as previous comment, but this time relating to `Aspa` types themselves rather than variant names of this type.

Reply:
I realize now that of course there is also the question of REST relative URL naming, and again for ROAs that was `/routes` but for ASPAs `/aspas` has been used, so same question here too.

Reply:
same answer..

Because many (authorized) 'routes' can end up on a single ROA. But an ASPA config maps straight to an ASPA object.